lib/fork_break/process.rb

```diff
@@ -4,16 +4,19 @@ class << self
       attr_accessor :breakpoint_setter
     end
 
+    attr_reader :return_value
+
     def initialize(debug = false, &block)
       @debug = debug
       @fork = Fork.new(:return, :to_fork, :from_fork) do |child_fork|
         self.class.breakpoint_setter = breakpoints = BreakpointSetter.new(child_fork, debug)
 
         breakpoints << :forkbreak_start
-        block.call(breakpoints)
+        returned_value = block.call(breakpoints)
         breakpoints << :forkbreak_end
```

Comment:
Can you test that?
From my experiences complex objects we're correctly marshalled between processes, since the parent process shares the same code with the child, but the only thing that did not survive are database connections.

So if you attempt to `.save` an ActiveRecord object it will probably fail.